src/js/events/ChaosEvents.js

```diff
@@ -0,0 +1,6 @@
+var ChaosEvents = {
+  FINISHED: "CHAOS_EVENTS_FINISHED",
+  STARTED: "CHAOS_EVENTS_STARTED"
```

Comment:
As we are now sending all task IDs at once to the endpoint, the 2 events are fired one right after each other (the endpoint simply tells us it accepted the task IDs to be killed). 

I have the feeling we should be getting back to this problem once we implement the `/events` endpoint stream – that way we could know about the actual state of the chaos/killing progress.